Automattic/jetpack
# Add mime `type` prop to custom media

When selecting a custom media for social posts, we only save `id` and `url` of the selected media to the post meta but in Calypso (See https://github.com/Automattic/wp-calypso/pull/82767), we need the media type information to generate the appropriate preview.

## Proposed changes:

* Add the `type` prop to the saved custom media
* Expose the prop via the registered meta

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Boot up the PR
* Add a new post
* Select a custom media for social post
* Confirm that `type` is saved to `additional_media` under `jetpack_social_options` meta
